docs/Getting Started Part 2/index.md

```diff
@@ -0,0 +1,323 @@
+  * Use the appropriate corresponding connection when sending responses and notifications to any connected component.
+
+!!!
+
+## Handshake
+For opening a WebSocket connection, a handshake must be performed.
+
+!!! info
+  1. Client/Server relationship
+    * SDL Core is the Server
+    * The HMI is the Client
+  2. Host
+    * SDL Core is listening on **127.0.0.1:8087** by default
+    * The IP and port is configurable in SDL Core's smartDeviceLink.ini file
+  3. WebSocket Protocol Version 13 is used by SDL Core
+
+!!!
+
+## Component Registration
+
+### Request
+
+The HMI must register each component which can communicate with SDL Core using the following RPC format
+
+| Key     | Value Info    |
+| :------------- | :------------- |
+| "id"       | a multiple of 100 (100, 200, 300, ...) |
+| "jsonrpc" | "2.0" - constant for all messages between SDL Core and the HMI|
+| "method" | "MB.registerComponent" - the request is assigned to SDL Core's MessageBroker where the component name will be associated with the socket ID. Further, SDL Core will send messages related to the named component over the corresponding connection|
+|"componentName"| The name of the component being registered. Must correspond to the appropriate component name described in the current guidelines.|
+
+Example Request:
+```json
+{
+  "jsonrpc": "2.0",
+  "id": 100,
+  "method": "MB.registerComponent",
+  "params": {
+    "componentName": "BasicCommunication"
+  }
+}
+```
+
+The possible componentNames are:  
+  * `BasicCommunication` - Generic interface responsible for a collection of information and user actions. Includes `UpdateAppList` and `OnAppRegistered`.
+  * `UI` - Interface responsible for RPC events and information made visible to the user. Includes `Show` and `Alert`. 
+  * `Buttons` - Interface responsible for RPC events and information related to hard and soft buttons in the vehicle. Includes `OnButtonPress` and `OnButtonEvent`.
+  * `VR` - Interface responsible for RPC events and information related to voice recognition. Contains RPC voice recognition counterparts to the UI component RPCs.
+  * `TTS` - Interface responsible for RPC events and information related to text to speech capabilities. Includes RPC `Speak`.
+  * `Navigation`- Interface responsible for RPC events and information related to navigation. Includes `StartStream` and `GetWayPoints`.
+  * `VehicleInfo`- Interface responsible for RPC events and information related to vehicle data. Includes `GetVehicleData` and `SubscribeVehicleData`.
+  * `RC` - Interface responsible for RPC events and information related to the [Remote Control Feature](https://github.com/smartdevicelink/sdl_evolution/blob/master/proposals/0065-remote-control.md).
+  * `AppService` - Interface responsible for RPC events and information related to the [App Services Feature](https://github.com/smartdevicelink/sdl_evolution/blob/master/proposals/0065-remote-control.md).
```

Comment:
both features link to remote control, also in the future should these link to documentation/guides over a proposal?

Reply:
Ill point it to the correct proposal, but yes in the future thats a good idea